apps/omg_eth/test/omg_eth/root_chain_test.exs

```diff
@@ -126,4 +121,56 @@ defmodule OMG.Eth.RootChainTest do
               ]} = deposits
     end
   end
+
+  describe "get_standard_exits_structs/2" do
```

Comment:
this is what you can cover with a end2end test in priv/

Reply:
Drafting as https://github.com/omisego/elixir-omg/pull/1243